src/jdk.jdi/share/native/libdt_shmem/shmemBase.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1999, 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1999, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The second year is 2020 in jdk17: I suggest it should be the same here.

Reply:
Updated them both.